fluentd/Dockerfile

```diff
@@ -14,21 +14,14 @@ LABEL io.k8s.description="Fluentd container for collecting of docker container l
   io.openshift.expose-services="9200:http, 9300:http" \
   io.openshift.tags="logging,elk,fluentd"
 
-RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
-  yum install -y --setopt=tsflags=nodocs \
-  gcc-c++ \
-  ruby \
-  ruby-devel \
-  libcurl-devel \
-  make && \
-  mkdir -p ${HOME} && \
-  gem install --no-ri fluentd fluent-plugin-kubernetes_metadata_filter fluent-plugin-elasticsearch fluent-plugin-flatten-hash && \
-  mkdir -p ${THROTTLE_CONF_LOCATION} && \
-  touch ${THROTTLE_CONF_LOCATION}/settings
-
 ADD fluent.conf /etc/fluent/fluent.conf
 ADD configs.d/ /etc/fluent/configs.d/
-ADD run.sh generate_throttle_configs.rb ${HOME}/
+ADD install.sh run.sh generate_throttle_configs.rb \
+    fluentd_es_config.conf fluentd_es_ops_config.conf \
```

Comment:
They are.  I just thought it would be more consistent to have all of the fluentd_es_*_config.conf files under $HOME, then copy from there to /etc/fluent/configs.d/output as needed.